cogs/wiki.py

```diff
@@ -25,6 +25,13 @@ def embed(self, title):
         embed.url = "https://wiki.ds-homebrew.com/"
         return embed
 
+    def skin_embed(self, title):
+        embed = discord.Embed(title=title)
+        embed.set_author(name="DS-Homebrew Wiki")
+        embed.set_thumbnail(url="https://avatars.githubusercontent.com/u/46971470?s=400&v=4")
+        embed.url = "https://skins.ds-homebrew.com/"
```

Comment:
Why not add an optional page (or some other name) arg or kwarg to the method instead? 
It would save you from doing `embed.url += "test"` a bunch of times.

Reply:
This idea was completely rewritten.